src/Label/Events/Event.php

```diff
@@ -5,7 +5,7 @@
 use Broadway\Serializer\SerializableInterface;
 use ValueObjects\Identity\UUID;
 
-abstract class AbstractEvent implements SerializableInterface
```

Comment:
Gewoon even meegeven dat we ook met abstractEvents werkten in de offer namespace. Daar maakten we dan een concrete implementatie van bij de tests. Die werd gebruikt om de abstract class te testen.